src/monodroid/jni/monodroid-glue.cc

```diff
@@ -634,6 +632,14 @@ JNI_OnLoad (JavaVM *vm, void *reserved)
 	env->DeleteLocalRef (System_javaVersion);
 	env->DeleteLocalRef (System_class);
 
+	if (utils.should_log (LOG_TIMING)) {
+		utils.current_time (total_time.end);
+
+		timing_diff diff;
```

Comment:
Why not make this work?

```c++
timing_diff diff = total_time.diff ();
```

Or something where we use an actual return value?

Reply:
Because then you always pay the price of invoking the constructor, whether you actually use the variable or not. With a POD it's just a stack space - practically no initialization cost time-wise.